Camera timeout (#132)
moaazsidat/react-native-qrcode-scanner: README.md

Use this to configure how long it should take before the `QRCodeScanner` should reactivate.

#### `cameraTimeout`
propType: `number`
default: `0`

Use this to configure how long it should take before the `QRCodeScanner` should be displayed. After that the camera will be inactive and press the view to reactivate it.

> The `0` default means it's always on.

Reviewer: maybe would be great to describe that the number expected in this property must to be in milliseconds, what do you think?

Author: Yes that would be good. I'll resubmit the PR

Use this to configure how long it should take before the `QRCodeScanner` should reactivate.

#### `cameraTimeout`
propType: `number`
default: `0`

Use this to configure how long it should take (in millisecond) before the `QRCodeScanner` should be displayed. After that the camera will be inactive and press the view to reactivate it.
The `0` default means it's always on.

#### `cameraTimeoutView`

propType: `element`

> Pass a RN element/component to use it when the camera is inactive (after `cameraTimeout` milliseconds).

Author: If the `cameraTimeoutView` is specified without `cameraTimeout`, the `cameraTimeoutView` is not going to be displayed in any circumstance. It will display only if you specify both props and after the camera has been activated for more than `cameraTimeout` milliseconds.

> Pass a RN element/component to use it when the camera is inactive (after `cameraTimeout` milliseconds).

Author: Many thanks to your review :) Should I put it like this

#### `cameraTimeoutView`

propType: `element`

Pass a RN element/component to show it when the camera is inactive for `cameraTimeout`  (another props) milliseconds. If the `camearTimeout` is `0` or not specified, this prop will never be used.

> Pass a RN element/component to use it when the camera is inactive (after `cameraTimeout` milliseconds).

Reviewer: > Pass a RN element/component to show it when the camera is inactive for cameraTimeout (another props) milliseconds. If the camearTimeout is 0 or not specified, this prop will never be used.

did you update the docs with this description 👆? it sounds pretty good to me 💯 

by the way, just fix the words typos 😆

> Pass a RN element/component to use it when the camera is inactive (after `cameraTimeout` milliseconds).

Author: Ok, I've pushed it. 

> by the way, just make sure that you will fix the words typos above before 😆

I seem don't see the typos you mentioned here 😟, would you mind tell me what went wrong.

> Pass a RN element/component to use it when the camera is inactive (after `cameraTimeout` milliseconds).

Reviewer: > I seem don't see the typos you mentioned here 😟, would you mind tell me what went wrong.

sure, no worries! I'll be put them as commit suggestion in review, sounds quickly to be solved, so closing this conversation

> Pass a RN element/component to show it when the camera is inactive for `cameraTimeout` (another props) milliseconds. If the `camearTimeout` is 0 or not specified, this prop will never be used.

Reviewer: ```suggestion
Pass an RN element/component to show it when the camera is inactive for `cameraTimeout` (another prop) milliseconds. If the `cameraTimeout` is 0 or not specified, this prop will never be used.
```
